script/GameObjects/Components/Core/Health.lua

```diff
@@ -59,6 +75,7 @@ function Entity:setHealth (value, max, rate)
 end
 
 function Entity:updateHealth (state)
-  if self:isDestroyed() then return end
-  self.health = min(self.healthMax, self.health + state.dt * self.healthRate)
+  if not self:isDestroyed() then
```

Comment:
Personally i´d prefer the old one liner approach, opening an if-statement feels a bit unecessary

Reply:
Future-proofing, that is. 😄 That area of the code is definitely going to change. When the game is close to stable and we do a maintenance cleanup pass through the code, one-liners are a good option.